src/react-app-env.d.ts

```diff
@@ -1,3 +1,19 @@
 /// <reference types="react-scripts" />
 
-type Todo = {}; /* TODO: DESCRIBE */
+export interface Todo {
+  id: number,
+  userId: number,
+  completed: boolean,
+  title: string,
+}
+
+export interface User {
+  id: number,
+  name: string,
+  username: string,
+  email: string,
+  phone: string,
+  website: string,
+  createdAt: string,
+  updatedAt: string,
+}
```

Comment:
много неиспользуемых полей
```suggestion
export interface User = {
  id: number;
  name: string;
  email: string;
  phone: string;
};
```